authorizations/authorization-jdbc/src/test/java/org/apache/gravitino/authorization/jdbc/JdbcAuthorizationPluginTest.java

```diff
@@ -0,0 +1,340 @@
+    expectNewOwners.clear();
+    currentIndex = 0;
+    expectSQLs = Lists.newArrayList("CREATE USER tmp");
+    currentSQLIndex = 0;
+
+    // Test catalog object
+    MetadataObject catalogObject = MetadataObjects.of(null, "catalog", MetadataObject.Type.CATALOG);
+    expectTypes.add(MetadataObject.Type.SCHEMA);
+    expectObjectNames.add("*");
+    expectPreOwners.add(Optional.empty());
+    expectNewOwners.add(owner);
+
+    expectTypes.add(MetadataObject.Type.TABLE);
+    expectObjectNames.add("*.*");
+    expectPreOwners.add(Optional.empty());
+    expectNewOwners.add(owner);
+    plugin.onOwnerSet(catalogObject, null, owner);
+
+    // clean up
+    expectTypes.clear();
+    expectObjectNames.clear();
+    expectPreOwners.clear();
+    expectNewOwners.clear();
+    currentIndex = 0;
+    expectSQLs = Lists.newArrayList("CREATE USER tmp");
+    currentSQLIndex = 0;
+
+    // Test schema object
+    MetadataObject schemaObject =
+        MetadataObjects.of("catalog", "schema", MetadataObject.Type.SCHEMA);
+    expectTypes.add(MetadataObject.Type.SCHEMA);
+    expectObjectNames.add("schema");
+    expectPreOwners.add(Optional.empty());
+    expectNewOwners.add(owner);
+
+    expectTypes.add(MetadataObject.Type.TABLE);
+    expectObjectNames.add("schema.*");
+    expectPreOwners.add(Optional.empty());
+    expectNewOwners.add(owner);
+    plugin.onOwnerSet(schemaObject, null, owner);
+
+    // clean up
+    expectTypes.clear();
+    expectObjectNames.clear();
+    expectPreOwners.clear();
+    expectNewOwners.clear();
+    currentIndex = 0;
+    expectSQLs = Lists.newArrayList("CREATE USER tmp");
+    currentSQLIndex = 0;
+
+    // Test table object
+    MetadataObject tableObject =
+        MetadataObjects.of(
+            Lists.newArrayList("catalog", "schema", "table"), MetadataObject.Type.TABLE);
+
+    expectTypes.add(MetadataObject.Type.TABLE);
+    expectObjectNames.add("schema.table");
+    expectPreOwners.add(Optional.empty());
+    expectNewOwners.add(owner);
+    plugin.onOwnerSet(tableObject, null, owner);
+  }
+
+  private static class TemporaryRole implements Role {
+    private final String name;
+
+    public TemporaryRole(String name) {
+      this.name = name;
+    }
+
+    @Override
+    public Audit auditInfo() {
+      return AuditInfo.EMPTY;
+    }
+
+    @Override
+    public String name() {
+      return name;
+    }
+
+    @Override
+    public Map<String, String> properties() {
+      return Collections.emptyMap();
+    }
+
+    @Override
+    public List<SecurableObject> securableObjects() {
+      return Collections.emptyList();
+    }
+  }
+
+  private static class TemporaryOwner implements Owner {
+    private final String name;
+    private final Type type;
+
+    public TemporaryOwner(String name, Type type) {
+      this.name = name;
+      this.type = type;
+    }
+
+    @Override
+    public String name() {
+      return name;
+    }
+
+    @Override
+    public Type type() {
+      return type;
+    }
+  }
```

Comment:
I think we can directly use `OwnerEnitity`, `RoleEntity` to instead `TemporaryRole ` and `TemporaryOwner `.

Reply:
We don't Owner entity.  So I just only replace `RoleEntity`.

Reply:
@jerqi Sorry, I mean is `GroupEntity` and `UserEntity`, So please change.

Reply:
Sorry, we can't. Because the plugin.onOwnerSet parameter is `Owner`